ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -204,10 +194,35 @@ function ComplianceLevelsByStandard() {
                 </Flex>
             }
         >
-            {complianceData && complianceData.length > 0 ? (
-                <ComplianceLevelsByStandardChart complianceData={complianceData} />
+            {Array.isArray(complianceData?.complianceLevelsByStandard) &&
+            complianceData.complianceLevelsByStandard.length !== 0 ? (
+                <ComplianceLevelsByStandardChart
+                    complianceLevelsByStandard={complianceData.complianceLevelsByStandard}
+                />
             ) : (
-                <ComplianceScanEmptyState />
+                <EmptyState className="pf-v5-u-h-100" variant="xs">
+                    <EmptyStateHeader
+                        titleText={
+                            Array.isArray(complianceData?.complianceStandards) &&
+                            complianceData.complianceStandards.length === 0
+                                ? 'No standards selected'
+                                : 'No results available.'
+                        }
+                        icon={<EmptyStateIcon className="pf-v5-u-font-size-xl" icon={SyncIcon} />}
+                        headingLevel="h3"
+                    />
+                    <EmptyStateBody>
+                        {Array.isArray(complianceData?.complianceStandards) &&
+                        complianceData.complianceStandards.length === 0
+                            ? 'Manage profiles on the Compliance page.'
+                            : 'Run a scan on the Compliance page.'}
+                    </EmptyStateBody>
+                    <EmptyStateFooter>
+                        <Button component={LinkShim} href={complianceBasePath}>
+                            Go to compliance
+                        </Button>
+                    </EmptyStateFooter>
+                </EmptyState>
```

Comment:
My initial thought here -- is this a good reason to add the `EmptyStateFooter` to our existing `EmptyStateTemplate` component?

Then a follow up thought - should we even use the `EmptyStateTemplate` anymore? It seems like it is logic-less and encapsulates 8 lines of PF component code, and then uses 3-8 lines of prop code at the call site. Would it be best to just use the PF provided components directly, like is done here?